### ISSUE TITLE
Crontab fix

### ISSUE THREAD

**Issue by svandragt** [svandragt/twelete#3]
Crontab doesn't seem to run correctly, I've added local diagnostics and will push (doc) fixes if required.

**Comment by svandragt** [comment]
This seems to work:
```
15 5 * * * cd /home/sander/src/twelete && /home/sander/.poetry/bin/poetry run python ./twelete.py >> /var/log/twelete.log 2>&1
```

**Comment by svandragt** [comment]
Fixed by ff7f8b6c59ada323c4dd7e4c7acc820acfef770e.
